draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -243,6 +243,25 @@ Any candidates exposed to the application via local descriptions MUST be
 identical to those provided during candidate gathering (i.e., MUST NOT
 contain private host IP addresses).
 
+#### mDNS registration and query answer
+
+As the mNDS name of any mDNS candidate is expected to be unique, the ICE agent
+SHOULD skip probing when registering the mDNS name. The ICE agent MAY skip
+sending a mDNS announcement for the mDNS name.
```

Comment:
Would be good to explain why the mDNS announcement is a MAY versus a SHOULD or SHOULD NOT.

Reply:
Added some more information